### PR TITLE
Update DG: add use case for vliste

### PR DESCRIPTION
**Pull request by ruo-x**
suggest to identify all event-volunteer use cases by UCEV<number> thoughts?

### CODE REVIEW

**Comment by zacwong2151** [comment]
> suggest to identify all event-volunteer use cases by UCEV thoughts?

hmm fair imo

**Comment by brein62** [comment]
i like how all of us decided to do use cases like rn cos i didn't realise u and max also did it

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -711,16 +711,37 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes from step 2.
 
-**Use case UCV04: Clear all volunteers in volunteer list**
+**Use case UCEV01: List all events joined by a volunteer**
```
Comment (brein62) [inline docs/DeveloperGuide.md:714]: should be `UCEV04` while max's suggestions are `UCEV01` - `UCEV03`

**Review by brein62 — CHANGES_REQUESTED** [review]
yeet

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -711,16 +711,37 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-*{More to be added}*
+**Extensions**
+
+* 1a. The volunteer list is empty.
+
+  Use case ends.
+
+* 1b. The input command is incorrect.
+
+    * 1a1. System prompts the user to provide a valid command.
```
Comment (tiongMax) [inline docs/DeveloperGuide.md:732]: shldn't it be 1b2 ?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (tiongMax) [reply docs/DeveloperGuide.md:714]: wait when did i suggested hahahha

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (ruo-x) [reply docs/DeveloperGuide.md:732]: nice catch! but shldnt it be 1b1

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (brein62) [reply docs/DeveloperGuide.md:714]: as in for ur ones change to UCEV01-UCEV03 (refer to my PR)

Comment (tiongMax) [reply docs/DeveloperGuide.md:714]: oh okeh

Comment (ruo-x) [reply docs/DeveloperGuide.md:714]: oki

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (ruo-x) [reply docs/DeveloperGuide.md:732]: nice catch ! it rly should be!

**Review by tiongMax — APPROVED** [review]
Lgtm!

**Review by brein62 — APPROVED** [review]
lookin awesome

**Pull request merged**